motion/ruby_motion_query/event.rb

```diff
@@ -32,7 +32,7 @@ def initialize(sender, event, block)
 
     def handle_gesture_or_event(gesture)
       # Don't fire :long_press events twice.
-      return if @sdk_event_or_recognizer == VIEW_GESTURES[:long_press] && gesture.state == UIGestureRecognizerStateEnded
+      return if gesture.class == VIEW_GESTURES[:long_press] && gesture.state != UIGestureRecognizerStateBegan
```

Comment:
Thanks @markrickert.  It looked fine to me I was just verifying it was as you said the same thing and not what caused the bug, to be aware of if we did the same in other areas, etc :)